gluon/restricted.py

```diff
@@ -199,7 +199,7 @@ def __str__(self):
 
 
 def compile2(code, layer):
-    return compile(code.rstrip(), layer, 'exec')
+    return compile(code, layer, 'exec')
```

Comment:
I will take your patch but the rstrip() is important because on windows the web editor has the bad habit of adding the \n\r at the end and compile(..) complains.